sql/core/src/main/scala/org/apache/spark/sql/Dataset.scala

```diff
@@ -2262,6 +2275,19 @@ class Dataset[T] private[sql](
   def distinct(): Dataset[T] = dropDuplicates()
 
   /**
+   * Returns a new [[Dataset]] that contains only the unique rows from this [[Dataset]], considering
+   * only the subset of columns. This is an alias for `dropDuplicates(cols)`.
+   *
+   * Note that, equality checking is performed directly on the encoded representation of the data
+   * and thus is not affected by a custom `equals` function defined on `T`.
+   *
+   * @group typedrel
+   * @since 2.0.0
+   */
+  @scala.annotation.varargs
+  def distinct(cols: String*): Dataset[T] = dropDuplicates(cols)
```

Comment:
Thank you always for fast feedbacks, @rxin . And for nice lunch. :)

Yes, right. For this, maybe it's not needed because `distinct` is usually used with `select`. 
Also, we can use `dropDuplicates` since it's just an alias of `dropDuplicates`.

I think `distinct` is a function name which is more consistent with SQL. If we have this, we can do this, too.
```
ds.select("_1", "_2", "_3").distinct("_1").orderBy("_1", "_2").show()
```